code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -47,6 +47,7 @@
 /mob/living/simple_animal/pet/cat/Initialize(mapload)
 	. = ..()
 	AddElement(/datum/element/pet_bonus, "purrs!")
+	GetComponent(/datum/component/pet_collar).has_resting_state = TRUE
```

Comment:
Okay this is definitely not good, prefer either a variable on wherever you're creating collars now, or make a subproc like `init_pet_collar()` that adds the component and gets overridden by cat.

Reply:
I've axed the component and it's part of both classes again.